ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -30,40 +33,89 @@ def data_store_group():
     default=False,
     help="Flag to REMOVE data from store subdirectories.",
 )
-def clean_orphaned_tables_command(column, remove):
-    for prefix in os.listdir(CLICKHOUSE_STORE_PATH):
-        path = CLICKHOUSE_STORE_PATH + "/" + prefix
-
-        process_path(path, prefix, column, remove)
-
-
-def process_path(path: str, prefix: str, column: str, remove: bool) -> None:
-    print(f"Processing path {path} with prefix {prefix}:")
+@option(
+    "--store-path",
+    "store_path",
+    default=CLICKHOUSE_PATH + "/store",
+    help="Set the store subdirectory path,  `/var/lib/clickhouse/store` by default.",
+)
+@option(
+    "--show-directory-size",
+    "show_directory_size",
+    is_flag=True,
+    default=False,
+    help="Flag to show metadata size.",
+)
+@option(
+    "--show-all-metadata",
+    "show_only_orphaned_metadata",
+    is_flag=True,
+    default=True,
+    help="Flag to only orphaned metadata.",
+)
+def clean_orphaned_tables_command(
+    ctx, column, remove, store_path, show_directory_size, show_only_orphaned_metadata
+):
+    results = []
+    for prefix in os.listdir(store_path):
+        path = store_path + "/" + prefix
+        path_result = process_path(path, prefix, column, remove, show_directory_size)
+        if show_only_orphaned_metadata and path_result["status"] != "not_used":
+            continue
+        if not show_directory_size:
+            del path_result["size"]
```

Comment:
When we don't want to see size in output ?

Reply:
Have removed the option. Seems like we always want to see sizes.